src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Extensions/StartupExtensions.cs

```diff
@@ -93,15 +93,13 @@ public static IServiceCollection AddAdventConfiguration(this IServiceCollection
 
         public static IServiceCollection RegisterGoogleHttpClient(this IServiceCollection services)
         {
-            var googleCalendarBaseUri = @"https://www.googleapis.com/calendar/v3/";
             services.AddHttpContextAccessor();
             services.AddHttpClient<IGoogleCalendarService, GoogleCalendarService>(
                 async (services, client) =>
                 {
                     var accessor = services.GetRequiredService<IHttpContextAccessor>();
                     var accessToken = await accessor.HttpContext.GetTokenAsync("Calendar", "access_token");
                     client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
-                    client.BaseAddress = new Uri(googleCalendarBaseUri);
```

Comment:
what about client.BaseAddress? is it not needed anymore?

Reply:
We are loading full URIs from appsettings and use them directly in the service